integration-cli/docker_cli_rename_test.go

```diff
@@ -95,18 +79,15 @@ func (s *DockerSuite) TestRenameCheckNames(c *check.C) {
 }
 
 func (s *DockerSuite) TestRenameInvalidName(c *check.C) {
-	runCmd := exec.Command(dockerBinary, "run", "--name", "myname", "-d", "busybox", "top")
-	if out, _, err := runCommandWithOutput(runCmd); err != nil {
+	if out, _, err := dockerCmdWithError("run", "--name", "myname", "-d", "busybox", "top"); err != nil {
```

Comment:
How come we're doing `dockerCmdWithError` in these cases where there is no need to inspect the error or output?